ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python3 --version
+pushd /tmp
+
+ls -la /root/kraken
+git clone https://github.com/redhat-chaos/krkn-hub.git
```

Comment:
we don't need to re-clone krkn-hub that folder structure/packages should already be there

You should add this to get data into es: https://github.com/openshift/release/blob/26858f402a2b1a3233c1c434ddb4bb9e933b9ffd/ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-commands.sh#L14C1-L19C1